cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/workflow/WorkflowStatus.java

```diff
@@ -25,12 +25,12 @@
 
   private  Service.State state;
   private  WorkflowActionSpecification currentAction;
```

Comment:
Yes. fork node does not have any action. 
`Map<NodeId, WorkflowStatus>` is returned while serving the `current` endpoint, which returns the specifications about the actions currently in execution. For `ACTION` node, the map would only have single entry. For `FORK` nodes the map can have multiple entries each representing the actions inside the fork, which are currently in execution.